src/SkyApm.Diagnostics.Logging.Serilog/SkyApm.Diagnostics.Logging.Serilog.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
In the csproj file, you need to import common.props , you can refer to [SkyApm.Diagnostics.SqlClient](https://github.com/SkyAPM/SkyAPM-dotnet/blob/master/src/SkyApm.Diagnostics.SqlClient/SkyApm.Diagnostics.SqlClient.csproj)